app/code/core/Mage/Core/Model/Design/Package.php

```diff
@@ -248,7 +248,7 @@ public function setTheme()
     {
         switch (func_num_args()) {
             case 1:
-                foreach (array('layout', 'template', 'skin', 'locale', 'default') as $type) {
+                foreach (['layout', 'template', 'skin', 'locale'] as $type) {
```

Comment:
The PR change is lost!

```suggestion
                foreach (['layout', 'template', 'skin', 'locale', 'default'] as $type) {
```